server/src/main/java/io/littlehorse/common/model/getable/core/wfrun/WfRunModel.java

```diff
@@ -68,6 +68,7 @@ public class WfRunModel extends CoreGetable<WfRun> {
     private WfRunIdModel id;
     private WfSpecIdModel wfSpecId;
     private List<WfSpecIdModel> oldWfSpecVersions = new ArrayList<>();
+    private int greatestThreadRunNumber;
```

Comment:
Somehow, this was added to the proto when I initially implemented ThreadRun retention, but I forgot to set it here.